uppsrc/CtrlCore/X11Ctrl.cpp

```diff
@@ -6,6 +6,7 @@ namespace Upp {
 
 void Ctrl::GuiPlatformConstruct()
 {
+	caretcx = caretcy = caretx = carety = 0; //aris002
```

Comment:
Rect should be constructed automatically.

Reply:
Same as https://github.com/ultimatepp/ultimatepp/pull/126#discussion_r1059698714